files/build_templates/init_cfg.json.j2

```diff
@@ -26,7 +26,7 @@
     },
     "CONTAINER_FEATURE": {
 {%- for container in ["bgp", "database", "dhcp_relay", "lldp", "pmon", "radv", "restapi", "sflow",
-                      "snmp", "swss", "syncd", "teamd", "telemetry"] %}
+                      "snmp", "swss", "syncd", "teamd", "telemetry", "nat"] %}
```

Comment:
Can you please place "nat" in alphabetical order amongst the others?

Reply:
done